NLog.Targets.OpenTelemetryProtocol/OtlpTarget.cs

```diff
@@ -173,6 +181,27 @@ private ResourceBuilder CreateResourceBuilder()
             return resourceBuilder;
         }
 
+        protected override void CloseTarget()
+        {
+            var logProvider = _loggerProvider;
+            _loggerProvider = null;
+            logProvider?.Dispose();
+            _loggers.Clear();
+
+            var processor = _processor;
+            _processor = null;
+            var result = processor?.Shutdown(1000) ?? true;
+            if (!result)
+                InternalLogger.Info("OtlpTarget(Name={0}) - Shutdown OpenTelemetry BatchProcessor unsuccesful");
```

Comment:
You are missing `{0}` as parameter. Maybe something like this:
```c#
InternalLogger.Info("{0} - Shutdown OpenTelemetry BatchProcessor unsuccesful", this);
```

Reply:
fixed